docs/PyPiPublishing.md

```diff
@@ -0,0 +1,159 @@
+[![Apache License Version 2.0](https://img.shields.io/badge/license-Apache_2.0-green.svg)](LICENSE)
+[![PyPI Downloads](https://static.pepy.tech/badge/nncf)](https://pypi.org/project/nncf/)
+
+# Neural Network Compression Framework (NNCF)
+
+Neural Network Compression Framework (NNCF) provides a suite of post-training
+and training-time algorithms for optimizing inference of neural networks in
+[OpenVINO&trade;](https://docs.openvino.ai) with a minimal accuracy drop.
+
+
+NNCF is designed to work with models from [PyTorch](https://pytorch.org/),
+[TensorFlow](https://www.tensorflow.org/), [ONNX](https://onnx.ai/) and
+[OpenVINO&trade;](https://docs.openvino.ai).
+
+The framework is organized as a Python package that can be built and used
+in a standalone mode. Its architecture is unified to make it easy to
+add different compression algorithms for both PyTorch and TensorFlow deep
+learning frameworks.
+
+
+NNCF provides [samples](#demos-tutorials-and-samples) that demonstrate the
+usage of compression algorithms for different use cases and models. See
+compression results achievable with the NNCF-powered samples on the
+[NNCF Model Zoo page](https://github.com/openvinotoolkit/nncf/blob/develop/docs/ModelZoo.md).
+
+For more information about NNCF, see:
+- [NNCF repository](https://github.com/openvinotoolkit/nncf)
+- [user documentation](https://docs.openvino.ai/nncf)
+- [NNCF API documentation](https://openvinotoolkit.github.io/nncf/autoapi/nncf/)
+- [usage examples](https://github.com/openvinotoolkit/nncf/tree/develop/docs/usage)
+- [Notebook tutorials](https://github.com/openvinotoolkit/openvino_notebooks/blob/latest/notebooks/README.md#model-training)
+
+
+
+
+## Table of contents:
+
+- [Key Features](#key-features)
+- [Installation](#installation-guide)
+- [Third-party integration](#third-party-repository-integration)
+- [NNCF Model Zoo](#nncf-compressed-model-zoo)
+
+
+
+## Key Features
+
+### Post-Training Compression Algorithms
+
+| Compression algorithm                                                                                                                                       | OpenVINO  | PyTorch   | TensorFlow    | ONNX          |
+| :---------------------------------------------------------------------------------------------------------------------------------------------------------- | :-------: | :-------: | :-----------: | :-----------: |
+| [Post-Training Quantization](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/post_training_compression/post_training_quantization/Usage.md) | Supported | Supported | Supported     | Supported     |
+| [Weights Compression](https://github.com/openvinotoolkit/nncf/blob/develop/docs/usage/post_training_compression/weights_compression/Usage.md)               | Supported | Supported | Not supported | Not supported |
```

Comment:
It looks like, key features is outdated relative to readme on develop. Please, update it.